### ISSUE TITLE
Uncaught 

### ISSUE THREAD

**Issue by MrYouzc** [naptha/tesseract.js#204]
With version 1.0.10
When using the Simple example of the documentation i got:
blob:http://localhost:3000/58a39e69-bed3-40f9-b568-ce206b4aea99:1 Uncaught 
importScripts("../node_modules/tesseract.js/dist/worker.js");Uncaught

**Comment by tradescantia** [comment]
I receive the same error. Any Ideas why?

**Comment by Theraghav** [comment]
Are you by any chance using node 8.9 and ng6.

**Comment by JshWrnr** [comment]
Getting this as well.

**Comment by tombalcaen** [comment]
Any news yet about this?

**Comment by fabriciohedlund** [comment]
I solved the same problem pointing the right path for tesseract.js importing.

Solution in my case:
import Tesseract from '../../../../node_modules/tesseract.js/dist/tesseract.js';

**Comment by pablopunk** [comment]
Did you solve it @MrYouzc ? I've tried @fabriciohedlund's solution but didn't work for me

**Comment by bmdproject99** [comment]
After considerable banging-head-against-wall, I now have tesseract.js working purely offline, using Node and Electron. I cleared cache in my electron app, disconnected internet, rebuilt the app, and verified that it works purely offline.

Here's what I did:

1.  I downloaded  tesseract.js   and  worker.js   from:
https://github.com/naptha/tesseract.js/tree/master/dist

I downloaded the core   index.js   from:
https://cdn.jsdelivr.net/gh/naptha/tesseract.js-core@0.1.0/index.js

I downloaded the lang file I need (English) from: 
https://github.com/tesseract-ocr/tessdata

I put all four files into /scripts


2. I edited  tesseract.js  as follows:  

In the spawnWorker function, I commented out all lines with Blob, leaving only
var worker = new Worker(workerOptions.workerPath).

See below (comments are mine):

exports.spawnWorker = function spawnWorker(instance, workerOptions) {
//    if (window.Blob && window.URL) {
//        var blob = new Blob(['importScripts("' + workerOptions.workerPath + '");']);
//        var worker = new Worker(window.URL.createObjectURL(blob));
//    } else {
        var worker = new Worker(workerOptions.workerPath);
//    }

    worker.onmessage = function (e) {
        var packet = e.data;
        instance._recv(packet);
    };
    return worker;
};

I suspect there is a CORS issue with Blob in Chomium.  Commenting out the lines removed the "Uncaught  Blob  file"  error I was getting.    Kudos to the following for suggesting this:
https://github.com/naptha/tesseract.js/issues/241#issuecomment-450621164


3.   I included the  script in my index.html page:

<script src='scripts/tesseract.js'></script>


4.  Here is the correct way to set the paths for  workerPath, langPath,  and corePath

(Note:  in my example, all files are in /scripts.  Adjust path accordingly).

I included the following at the top of my script page:

window.Tesseract = Tesseract.create({
    workerPath: './scripts/worker.js',
    langPath: '',
    corePath: 'index.js'
})

It turns out langPath and corePath  are  relative to  worker.js!!!  

I hope this helps.
